### ISSUE TITLE
0.0.10 Sound and Mic stop working after a few minutes

### ISSUE THREAD

**Issue by RickRNF** [crmarsh/discord-linux-bugs#120]
In Ubuntu 16.04 on 0.0.10 I am unable to hear anyone and they are unable to hear me after a few minutes of being in a voice channel. 

I see my avatar light up as if it were picking up sound, but no one can hear me. My friend said his avatar would light up picking up sound fine, but I didn't see it light up at all.

There was no problem in 0.0.9 as far as I could tell, but I only used it for a day. 

I reinstalled 0.0.8 and there are no issues.

**Comment by ahjolinna** [comment]
I notice this also (on chakraOS using KDE), for me and my friend the mic stopped working everytime if you adjusted the volume in system instead of discord.

also for some reason I see "WEBRTC VoiceEngine: playStream" name instead of Discord on my plasma-pa/volume settings.

**Comment by RickRNF** [comment]
I don't think it happened for me from adjusting volume since I don't remember adjusting it. It might be one of the causes though, I'll try it out tonight.

**Comment by RickRNF** [comment]
I have been using 0.0.11 since it came out and the problem happened once or twice the first day, but no problems since. I will post if it happens again.

**Comment by ggdaynight** [comment]
I may have found the source of the problem. If the "Enable Quality of Service High Packet Priority" is turned on, on the other side of the voice conversation won't hear you. If you turn that feature off, than your sound is going through.

**Comment by OvermindDL1** [comment]
> I may have found the source of the problem. If the "Enable Quality of Service High Packet Priority" is turned on, on the other side of the voice conversation won't hear you. If you turn that feature off, than your sound is going through.

That there sounds like a broken router?

**Comment by ggdaynight** [comment]
I don't think it's broken, because the router is used in an everyday matter, and never experienced any kind of issue like that. Multiple VoIP based applications are used through that router as well.

**Comment by OvermindDL1** [comment]
> I don't think it's broken, because the router is used in an everyday matter, and never experienced any kind of issue like that. Multiple VoIP based applications are used through that router as well.

You'd be surprised, basic consumer routers are not at all reliable, even though they may seem that way 'on average', they are entirely full of bugs (that I have to deal with at work routinely).  ^.^;
